gatsby-config.js

```diff
@@ -0,0 +1,50 @@
+const fs = require("fs");
+const pluralize = require("pluralize");
+const workingGroups = JSON.parse(
+  fs.readFileSync(require.resolve("./src/data/working-groups.json"))
+);
+
+module.exports = {
+  siteMetadata: {
+    title: "EI2030",
+  },
+  pathPrefix: "/EI2030",
+  plugins: [
+    "gatsby-plugin-theme-ui",
+    "gatsby-transformer-remark",
+    {
+      resolve: "gatsby-transformer-json",
+      options: {
+        typeName: ({ node }) =>
+          node.name
+            .split("-")
+            .map((w) => pluralize.singular(w.toLowerCase()))
+            .map((w) => w.charAt(0).toUpperCase() + w.slice(1))
+            .join(""),
+      },
+    },
+    {
+      resolve: "gatsby-source-filesystem",
+      options: {
+        name: "pages",
+        path: "./src/pages/",
+      },
+      __key: "pages",
+    },
+    {
+      resolve: "gatsby-source-filesystem",
+      options: {
+        path: "./src/data/",
+      },
+    },
+    ...workingGroups.map(({ name, gitRemote }) => ({
+      resolve: "gatsby-source-git",
+      options: {
+        name,
+        branch: "main",
+        patterns: "website/**",
+        remote: gitRemote,
+      },
+    })),
```

Comment:
Here we are using the contents of the `working-groups.json` to dynamically define our "git sources".  You'll note that there are certain properties that are statically defined like `patterns: "website/**"`.  This informs the plugin what subdirectory to go looking for content inside of.  

So the assumption here is that all working group repositories will have a subdirectory named `website/` and that we'll pull in all markdown from there.

Reply:
@schickm This looks great. I like the use of the `working-groups.json` to define our git sources, the properties defining the branch, pattern, and remote.